jupyter_scheduler/executors.py

```diff
@@ -126,13 +128,16 @@ def execute(self):
             store_widget_state=True,
         )
 
-        ep.preprocess(nb)
-
-        for output_format in job.output_formats:
-            cls = nbconvert.get_exporter(output_format)
-            output, resources = cls().from_notebook_node(nb)
-            with fsspec.open(self.staging_paths[output_format], "w", encoding="utf-8") as f:
-                f.write(output)
+        try:
+            ep.preprocess(nb)
+        except CellExecutionError as e:
```

Comment:
Looks like you are also solving the issue related to cell executions errors being treated as job failures.